Library/Homebrew/test/utils_spec.rb

```diff
@@ -58,4 +58,41 @@
       expect(described_class.pluralize("foo", 2, singular: "o", plural: "es")).to eq("fooes")
     end
   end
+
+  describe ".underscore" do
+    # commented out entries require acronyms inflections
+    let(:words) {
+      [
+        ["API", "api"],
+        ["APIController", "api_controller"],
+        ["Nokogiri::HTML", "nokogiri/html"],
+        # ["HTTPAPI", "http_api"],
+        ["HTTP::Get", "http/get"],
+        ["SSLError", "ssl_error"],
+        # ["RESTful", "restful"],
```

Comment:
I believe these examples already don't work (trying them in `brew irb` on master), so this shouldn't be a breaking change, just highlighting where we aren't matching the ActiveSupport test suite.